bench/db_bench.ml

```diff
@@ -110,91 +124,88 @@ module Index = struct
 
   let read r () = Array.iter (fun (k, _) -> ignore (Index.find r k)) random
 
-  let write_random () =
+  let write_random name =
     Index_unix.reset_stats ();
-    let rw = Index.v ~fresh:true ~log_size (root // "fill_random") in
-    print_results (write rw) nb_entries;
+    let rw = Index.v ~fresh:true ~log_size (root // name) in
     write_amplif ();
-    rw
+    (rw, [ print_results (write rw) nb_entries name ])
```

Comment:
I think you don't need to return a list here, but directly the results of `print_results`, since you always return a list of one element. This way you don't need to call `list.flatten` at the end either